tests/lib/rules/no-deprecated.js

```diff
@@ -295,21 +283,15 @@ ruleTester.run('no-deprecated', rule, {
       `,
       errors: [
         errorMessage(
-          'componentWillMount', '16.9.0', 'UNSAFE_componentWillMount',
-          'https://reactjs.org/docs/react-component.html#unsafe_componentwillmount. '
-            + 'Use https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles to automatically update your components.',
-          { type: 'Identifier', line: 3, column: 11 }
+          'componentWillMount', '16.9.0', 'UNSAFE_componentWillMount', 'https://reactjs.org/docs/react-component.html#unsafe_componentwillmount. '
+            + 'Use https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles to automatically update your components.', { type: 'Identifier', line: 3, column: 11 }
         ),
         errorMessage(
-          'componentWillReceiveProps', '16.9.0', 'UNSAFE_componentWillReceiveProps',
-          'https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops. '
-            + 'Use https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles to automatically update your components.',
-          { type: 'Identifier', line: 4, column: 11 }
+          'componentWillReceiveProps', '16.9.0', 'UNSAFE_componentWillReceiveProps', 'https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops. '
+            + 'Use https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles to automatically update your components.', { type: 'Identifier', line: 4, column: 11 }
         ),
-        errorMessage('componentWillUpdate', '16.9.0', 'UNSAFE_componentWillUpdate',
-          'https://reactjs.org/docs/react-component.html#unsafe_componentwillupdate. '
-            + 'Use https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles to automatically update your components.',
-          { type: 'Identifier', line: 5, column: 11 }
+        errorMessage('componentWillUpdate', '16.9.0', 'UNSAFE_componentWillUpdate', 'https://reactjs.org/docs/react-component.html#unsafe_componentwillupdate. '
+            + 'Use https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles to automatically update your components.', { type: 'Identifier', line: 5, column: 11 }
```

Comment:
This is not the ideal fix here; since the entire function call doesn't fit on one line, each argument must appear on a line by itself.

I'll update these later today, if you don't get to it first.

Reply:
This was changed automatically when running `eslint --fix`

Reply:
Sure, but autofix output isn’t the only (or the best) way to satisfy eslint rules :-)